lib/Driver/ToolChains.cpp

```diff
@@ -1286,6 +1300,24 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   if (!Linker.empty()) {
     Arguments.push_back(context.Args.MakeArgString("-fuse-ld=" + Linker));
   }
+  
+  // Configure the toolchain
+  // By default, use the system clang++ to link.
+  const char * Clang = "clang++";
```

Comment:
Sure, Im thinking about the default case, where no `-tools-directory` is passed.